src/systems/filecoin_vm/runtime/impl/runtime.go

```diff
@@ -636,16 +640,8 @@ func (rt *VMContext) ValueReceived() actors.TokenAmount {
 	return rt._valueReceived
 }
 
-func (rt *VMContext) Randomness(tag filcrypto.DomainSeparationTag, epoch block.ChainEpoch) util.Randomness {
-	IMPL_TODO()
-	panic("")
-}
-
-func (rt *VMContext) RandomnessWithAuxSeed(
-	tag filcrypto.DomainSeparationTag, epoch block.ChainEpoch, auxSeed util.Serialization) util.Randomness {
-
-	IMPL_TODO()
-	panic("")
+func (rt *VMContext) Chain() chain.Chain {
```

Comment:
It looks like this is never called, but GetRandomness is unimplemented, but should be done so in terms of chain.